_posts/en/newsletters/2023-09-13-newsletter.md

```diff
@@ -0,0 +1,181 @@
+  bips] seven draft BIPs---one more than in the initial announcement of
+  the protocol, then under the name _Taro_ (see [Newsletter
+  #195][news195 taro]).  To the Lightning-Dev mailing list, he
+  [announced][osuntokun blip post] a [draft BLIP][osuntokun blip] for
+  spending and receiving taproot assets using LN, with the protocol
+  based on the experimental "simple taproot channels" feature planned to
+  be released in LND 0.17.0-beta.
+
+    Note that, despite its name, Taproot Assets is not part of the
+    Bitcoin Protocol and does not change the consensus protocol in any
+    way.  It uses existing capabilities to provide new features for
+    users that opt-in to its client protocol.
+
+    None of the specifications had received any discussion on the
+    mailing list as of this writing.
+
+- **LN messaging changes for PTLCs:** as the first LN implementation
+  with experimental support for channels using [P2TR][topic taproot]
+  and [MuSig2][topic musig] is expected to be released soon, Greg
+  Sanders [posted][sanders post] to the Lightning-Dev mailing list a
+  [summary][sanders ptlc] of several different previously-discussed
+  changes to LN messages to allow them to support sending payments with
+  [PTLCs][topic ptlc] instead of [HTLCs][topic htlc].  For most
+  approaches, the changes to messages do not seem large or invasive,
+  but we note that most implementations will probably continue using one
+  set of messages for handling legacy HTLC forwarding while also
+  offering upgraded messages to support PTLC forwarding, creating two
+  different paths that will need to be maintained concurrently until
+  HTLCs are phased out.  If some implementations add experimental PTLCs
+  support before the messages are standardized, then implementations
+  might even be required to support three or more different protocols
+  simultaneously, to the disadvantage of all.
+
+    Sander's summary had not received any comments as of this writing.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review
+Club][] meeting, highlighting some of the important questions and
+answers.  Click on a question below to see a summary of the answer from
+the meeting.*
+
+[transport abstraction][review club 28165] is a PR by Pieter Wuille (sipa)
+that introduces a _transport_ absraction (interface class). Concrete
+derivations of this class convert a (per-peer) connection's (already
+serialized) send and receive messages to and from wire format. This
+can be thought of as implementing a deeper level of serialization and
+deserialization. These classes do not do the actual sending and receiving.
+
+The PR derives two concrete classes from the `Transport` class,
+`V1transport` (what we have today) and `V2Transport` (encrypted on the
+wire). This PR, which was recently merged, is part of the
+[BIP324][bip-transport-v2]
+_Version 2 P2P Encrypted Transport Protocol_ project.
```

Comment:
Suggest a topic link to https://bitcoinops.org/en/topics/v2-p2p-transport/

Reply:
Might also link part of the text ("project"?) to the tracking issue: https://github.com/bitcoin/bitcoin/issues/27634